<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QToolbar</class>
 <widget class="QWidget" name="QToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>48</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="auto_scale_button">
       <property name="text">
        <string>autoscale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="center_button">
       <property name="text">
        <string>center</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="panzoom_button">
       <property name="text">
        <string>panzoom</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="maintain_aspect_button">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1:1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="y_direction_button">
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_polygon_button">
       <property name="text">
        <string>polygon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="record_button">
       <property name="text">
        <string>record</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
